### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by rabalyn**
Hi,

Disclaimer:
...this is my first PR on github, so feedback is welcome if I am doing something really bad here ;-)

I am currently starting with feathers and wanted to integrate a file upload service to my project. The official docs are refering to this repository - and I thought it might be nice to update the dependencies and devDependencies :-)

While fixing the broken dependency usage I replaced the `var`s with `const`s.

Tests are working.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,33 +20,35 @@ var listen = function(stream, opts, cb) {
...
 
-  var proxy = listen(duplexify(), opts, cb)
+  const proxy = listen(duplexify(), opts, cb)
 
   proxy.setReadable(false)
 
-  mkdirp(dir, function(err) {
-    if (proxy.destroyed) return
-    if (err) return proxy.destroy(err)
+  mkdirp(dir)
```
Comment (mafintosh) [inline index.js]: Can use mkdirp-classic instead?

**Comment by mafintosh** [comment]
Thanks! Added a comment. Can you also remove the lock file? :)

File: index.js (hunk 1 above)
Comment (rabalyn) [reply index.js]: Yes, of course - I made the changes :-)

Comment (mafintosh) [reply index.js]: 👍

**Comment by mafintosh** [comment]
Woooooot! 6.0.0 :)

**Pull request merged**